### PR TITLE
docs: Update team responsibilities

### PR DESCRIPTION
**Pull request by nzakas**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)

Updated governance docs to make explicit the team member responsibilities discussed internally.

Also deleted the processes for adding different team members as they are handled in our private repo with issue templates.

### CODE REVIEW

**Review by harish-sethuraman — APPROVED** [review]
LGTM

**Comment by christian-bromann** [comment]
@nzakas 👀

**Comment by fasttime** [comment]
The browser test is passing again now 😊

**Review by aladdin-add — APPROVED** [review]

File: docs/src/contribute/governance.md [hunk 1]
```diff
@@ -55,12 +60,19 @@ Committers are community members who have shown that they are committed to the c
 
 Committers:
 
+* Are expected to work at least one hour per week triaging issues and reviewing pull requests.
+* Are expected to work at least two hours total per week on ESLint.
+* May invoice for the hours they spend working on ESLint at a rate of $50 USD per hour.
+* Are expected to check in on Discord once per week day for team updates.
 * Are expected to work on public branches of the source repository and submit pull requests from that branch to the master branch.
 * Are expected to delete their public branches when they are no longer necessary.
+* Are expected to provided feedback on issues in the "Feedback Needed" column of the [Triage Board](https://github.com/orgs/eslint/projects/3/views/1).
```
Comment (fasttime) [inline docs/src/contribute/governance.md:69]:
```suggestion
* Are expected to provide feedback on issues in the "Feedback Needed" column of the [Triage Board](https://github.com/orgs/eslint/projects/3/views/1).
```

File: docs/src/contribute/governance.md [hunk 2]
```diff
@@ -32,12 +32,17 @@ Website Team Members are community members who have shown that they are committe
 
  Website Team Members:
 
+* Are expected to work at least one hour per week triaging issues and reviewing pull requests.
+* Are expected to work at least two hours total per week on ESLint.
+* May invoice for the hours they spend working on ESLint at a rate of $50 USD per hour.
+* Are expected to check in on Discord once per week day for team updates.
```
Comment (Rec0iL99) [inline docs/src/contribute/governance.md:38]: I think mentioning the `#team` Discord channel here would be a good idea.

File: docs/src/contribute/governance.md [hunk 3]
```diff
@@ -32,12 +32,17 @@ Website Team Members are community members who have shown that they are committe
...
 * Are expected to work on public branches of the source repository and submit pull requests from that branch to the master branch.
 * Are expected to delete their public branches when they are no longer necessary.
 * Must submit pull requests for all changes.
 * Have their work reviewed by Reviewers and TSC members before acceptance into the repository.
 * May label and close website-related issues (see [Manage Issues](../maintain/manage-issues))
 * May merge some pull requests (see [Review Pull Requests](../maintain/review-pull-requests))
+* Are expected to notify the team in Discord if they will be unavailable for more than a few days.
```
Comment (Rec0iL99) [inline docs/src/contribute/governance.md:45]: Here too.

File: docs/src/contribute/governance.md [hunk 4]
```diff
@@ -55,12 +60,19 @@ Committers are community members who have shown that they are committed to the c
 
 Committers:
 
+* Are expected to work at least one hour per week triaging issues and reviewing pull requests.
+* Are expected to work at least two hours total per week on ESLint.
+* May invoice for the hours they spend working on ESLint at a rate of $50 USD per hour.
+* Are expected to check in on Discord once per week day for team updates.
```
Comment (Rec0iL99) [inline docs/src/contribute/governance.md:66]: Here too.

File: docs/src/contribute/governance.md [hunk 5]
```diff
@@ -55,12 +60,19 @@ Committers are community members who have shown that they are committed to the c
...
 * Are expected to work on public branches of the source repository and submit pull requests from that branch to the master branch.
 * Are expected to delete their public branches when they are no longer necessary.
+* Are expected to provided feedback on issues in the "Feedback Needed" column of the [Triage Board](https://github.com/orgs/eslint/projects/3/views/1).
+* Are expected to work on at least one issue in the "Ready to Implement" column of the [Triage Board](https://github.com/orgs/eslint/projects/3/views/1) that they didn't create each month.
 * Must submit pull requests for all changes.
 * Have their work reviewed by TSC members before acceptance into the repository.
 * May label and close issues (see [Manage Issues](../maintain/manage-issues))
 * May merge some pull requests (see [Review Pull Requests](../maintain/review-pull-requests))
+* Are expected to notify the team in Discord if they will be unavailable for more than a few days.
```
Comment (Rec0iL99) [inline docs/src/contribute/governance.md:75]: Here too.

File: docs/src/contribute/governance.md (hunk 2 above)
Comment (nzakas) [reply docs/src/contribute/governance.md:38]: I wasn't sure if I should do that because `#team` isn't public, but this document is.... :smile:

But since it's mentioned in this comment, I suppose not a big deal to add it.

File: docs/src/contribute/governance.md [hunk 6]
```diff
@@ -32,12 +32,17 @@ Website Team Members are community members who have shown that they are committe
 
  Website Team Members:
 
+* Are expected to work at least one hour per week triaging issues and reviewing pull requests.
+* Are expected to work at least two hours total per week on ESLint.
+* May invoice for the hours they spend working on ESLint at a rate of $50 USD per hour.
+* Are expected to check in on the `#team` Discord channel once per week day for team updates.
```
Comment (JoshuaKGoldberg) [inline docs/src/contribute/governance.md:38]: This assumes folks never take any week days off. What kind of time off is acceptable?

Proposal: maybe rephrase this to say something like "per _working_ day", and add a list item about expectations of working most weeks in a month?

_(here and for committers)_

**Review by JoshuaKGoldberg — COMMENTED** [review]
Seems reasonable per internal discussions. I do think it'd be good to iterate on the _"once per week day"_ point - thoughts?

File: docs/src/contribute/governance.md (hunk 6 above)
Comment (nzakas) [reply docs/src/contribute/governance.md:38]: We don't have any specific expectations around working weeks in a month. Would that be helpful for you? 

I'm not sure what the definition of "working day" is, so I fear that would introduce more confusion. Maybe "week days (excluding local holidays and other time off)"?

The point I'm trying to make here is that we need people active during the week but not on the weekends (unless you want to).

Maybe we can update the bullet about letting us know about time off with something like "You are free to take time off whenever you want, just post in `#team` if you're going to be away for more than a couple of days."?

Comment (JoshuaKGoldberg) [reply docs/src/contribute/governance.md:38]: Yeah that makes sense. I think it's generally hard to be precise here. 👍 from me on either or both of those changes.

Comment (nzakas) [reply docs/src/contribute/governance.md]: Okay, see if this latest wording helps.

**Review by JoshuaKGoldberg — APPROVED** [review]
💯

**Review by fasttime — APPROVED** [review]

**Review by amareshsm — APPROVED** [review]
LGTM 🎉
